anime_downloader/sites/anime.py

```diff
@@ -453,13 +453,16 @@ class SearchResult:
         URL for the poster of the anime.
     meta: dict
         Additional metadata regarding the anime.
+    meta_info: dict
+        Metadata regarding the anime. Not shown in the results, used to match with MAL
     """
 
-    def __init__(self, title, url, poster='', meta=''):
+    def __init__(self, title, url, poster='', meta='', meta_info={}):
```

Comment:
Needed to provide more metadata without impacting the search results showed to the user.